### PR TITLE
UI Refactor for Authentication Screens

### PR DESCRIPTION
**Pull request by aminebenji04**
This PR focuses on UI updates and refinements for the authentication screens (sign-in and sign-up) to align them with the Figma mock-ups and improve the overall visual consistency.
### UI Enhancements:
- Refactored text fields to enhance their visuals and maintain consistency across screens.
- Adjusted the sign-in screen to adhere strictly to Figma mock-ups.
- Converted the sign-in button into a card to ensure coherence among elements.
- Updated the sign-up screen UI to follow the style of the sign-in screen.

### CODE REVIEW

File: app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt
Comment (Yassyass181104) [inline app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt:1]: Good refactoring !

File: app/src/main/java/com/android/sample/ui/authentication/SignIn.kt [hunk 1]
```diff
@@ -102,52 +100,70 @@ fun SignInScreen(navigationActions: NavigationActions, viewModel: SignInViewMode
...
+                                      passwordState.value,
+                                      onProfileExists,
+                                      onProfileMissing,
+                                      onSignInFailure)
+                                }
+                              }
+                              Log.d("SignInScreen", "Sign in with email/password")
+                            })
+                      },
+                      modifier = Modifier.fillMaxWidth().height(60.dp).testTag("SignInButton"),
```
Comment (Yassyass181104) [inline app/src/main/java/com/android/sample/ui/authentication/SignIn.kt:149]: Here you have some magic values, please correct this thanks

**Review by Yassyass181104 — CHANGES_REQUESTED** [review]
As I can see and try to imagine in my head haha all looks good except some parts of the code where there are some magic values. Everything is clean except what I pointed out, good job !

File: app/src/main/java/com/android/sample/ui/authentication/SignUp.kt [hunk 2]
```diff
@@ -93,42 +103,55 @@ fun SignUpScreen(navigationActions: NavigationActions) {
...
+                      else -> {
+                        createUserWithEmailAndPassword(
+                            emailState.value,
+                            passwordState.value,
+                            context,
+                            onSuccess = { navigationActions.navigateTo(Screen.CREATE_PROFILE) })
+                      }
                     }
+                  },
+                  shape = RoundedCornerShape(12.dp),
```
[the remark this answers is not in the document]
Comment (aminebenji04) [reply app/src/main/java/com/android/sample/ui/authentication/SignUp.kt]: It has been resolved now

File: app/src/main/java/com/android/sample/ui/authentication/SignIn.kt [hunk 3]
```diff
@@ -236,31 +255,29 @@ fun rememberGoogleSignInIntent(context: Context, token: String): Intent {
...
-                  contentDescription = "Google Logo",
-                  modifier = Modifier.size(IMAGE_SIZE.dp).padding(end = STANDARD_PADDING.dp))
-              Text(
-                  text = "Sign in with Google",
-                  color = Color.Gray,
-                  fontSize = MEDIUM_PADDING.sp,
-                  fontWeight = FontWeight.Medium)
-            }
-      }
+  Card(
```
[the remark this answers is not in the document]
Comment (aminebenji04) [reply app/src/main/java/com/android/sample/ui/authentication/SignIn.kt:263]: Done

File: app/src/main/java/com/android/sample/ui/authentication/SignIn.kt (hunk 1 above)
Comment (aminebenji04) [reply app/src/main/java/com/android/sample/ui/authentication/SignIn.kt]: Done

File: app/src/main/java/com/android/sample/ui/authentication/SignIn.kt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Yassyass181104) [reply app/src/main/java/com/android/sample/ui/authentication/SignIn.kt:267]: There are still some magic values here

**Review by Yassyass181104 — APPROVED** [review]
Perfect, you corrected some little imperfections now it's all good

**Pull request merged**